File: socialuni-sdk/src/main/java/com/socialuni/sdk/model/RO/talk/circle/SocialCircleRO.java
package com.socialuni.sdk.model.RO.talk.circle;

import lombok.Data;

/**
 * @author qinkaiyuan
 * @date 2019-11-07 15:20
 */

@Data
public class SocialCircleRO {
    public Integer id;
    public String name;
    public String avatar;
    public String description;
    public Boolean selected;
    private Integer talkCount;
    private Integer count;
}

File: socialuni-js/packages/socialuni-api/src/model/community/circle/SocialCircleRO.ts
export default class SocialCircleRO {
  public id: number = null
  public name: string = null
  public selected = false
  public count: number = null
  public talkCount: number = null
  public avatar: string = null
  public description: string = null


  constructor (name?: string) {
    this.name = name || null
  }
}
